pkgs/data/fonts/oxygenfonts/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitHub }:
+
+let
+  pname = "oxygenfonts";
+  version = "unstable-20140213";
```

Comment:
I think it is fine to simply use `version = "20140213"`, i.e., drop the unstable part.